packages/genesys-spark-components/src/components/beta/gux-link/gux-link.scss

```diff
@@ -0,0 +1,112 @@
+ *  This global styling should also be removed in v5
+ *  Jira ticket for that: https://inindca.atlassian.net/browse/COMUI-3265
+ */
+
+:host {
+  padding-block: var(--gse-ui-links-inLine-padding);
+
+  ::slotted(a) {
+    inline-size: fit-content;
+    font-family: var(--gse-ui-links-inLine-medium-text-fontFamily),
+      var(--gse-semantic-theme-fontFamily-body), sans-serif;
+    font-size: var(--gse-ui-links-inLine-medium-text-fontSize);
+    font-weight: var(--gse-ui-links-inLine-medium-text-fontWeight);
+    line-height: var(--gse-ui-links-inLine-medium-text-lineHeight);
+    color: var(--gse-ui-links-default-foregroundColor) !important;
+    border-radius: var(--gse-ui-links-borderRadius);
+  }
+
+  ::slotted(a:hover) {
+    color: var(--gse-ui-links-hover-foregroundColor) !important;
+  }
+
+  ::slotted(a:active) {
+    color: var(--gse-ui-links-active-foregroundColor) !important;
+  }
+
+  ::slotted(a:visited) {
+    color: var(--gse-ui-links-visited-foregroundColor) !important;
+  }
+
+  ::slotted(a:focus-visible) {
+    @include focus.gux-focus-ring;
+  }
+}
+
+:host([size='small']) {
+  ::slotted(a) {
+    font-family: var(--gse-ui-links-inLine-small-text-fontFamily),
+      var(--gse-semantic-theme-fontFamily-body), sans-serif;
+    font-size: var(--gse-ui-links-inLine-small-text-fontSize);
+    font-weight: var(--gse-ui-links-inLine-small-text-fontWeight);
+    line-height: var(--gse-ui-links-inLine-small-text-lineHeight);
+  }
+}
+
+:host([standalone]) {
+  display: block;
+  padding-block: var(--gse-ui-links-standalone-padding);
+
+  ::slotted(a) {
+    display: flex;
+    gap: var(--gse-ui-links-standalone-gap);
+    align-items: center;
+    inline-size: fit-content;
+    font-family: var(--gse-ui-links-standalone-medium-text-fontFamily),
+      var(--gse-semantic-theme-fontFamily-body), sans-serif;
+    font-size: var(--gse-ui-links-standalone-medium-text-fontSize);
+    font-weight: var(--gse-ui-links-standalone-medium-text-fontWeight);
+    line-height: var(--gse-ui-links-standalone-medium-text-lineHeight);
+    text-decoration: none !important;
+  }
+
+  ::slotted(a:hover) {
+    font-family: var(--gse-ui-links-standalone-medium-underlinedText-fontFamily),
+      var(--gse-semantic-theme-fontFamily-body), sans-serif;
+    font-size: var(--gse-ui-links-standalone-medium-underlinedText-fontSize);
+    font-weight: var(
+      --gse-ui-links-standalone-medium-underlinedText-fontWeight
+    );
+    line-height: var(
+      --gse-ui-links-standalone-medium-underlinedText-lineHeight
+    );
+    text-decoration: var(
+      --gse-ui-links-standalone-medium-underlinedText-textDecoration
+    ) !important;
+  }
+
+  ::slotted(a:active) {
+    text-decoration: none !important;
+  }
+}
+
+:host([size='small'][standalone]) {
+  ::slotted(a) {
+    font-family: var(--gse-ui-links-standalone-small-text-fontFamily),
+      var(--gse-semantic-theme-fontFamily-body), sans-serif;
```

Comment:
does sans-serif need a token.

Reply:
iirc sans-serif is the fallback if the two other tokens fail to resolve for some reason. The second token is already a fallback. I took it from the global classes we already have and just changed the first token